api/XMLHttpRequest.json

```diff
@@ -1112,9 +1112,15 @@
             "edge_mobile": {
               "version_added": true
             },
-            "firefox": {
-              "version_added": true
-            },
+            "firefox": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "30",
+                "notes": "Starting in Firefox 30, synchronous requests on the main thread have been deprecated due to their negative impact on performance and the user experience. Therefore, the <code>async</code> parameter may not be <code>false</code> except in a <code>Worker</code>."
```

Comment:
This should be a note of the previous one.
"notes" is not an identifying entry in the schema. So in the way you describe it, you are saying that open has been added in Firefox 30, which is not true.